tests/server/routes/members-only-test.js

```diff
@@ -7,7 +7,7 @@ var db = require('../../../server/db');
 
 var supertest = require('supertest');
 
-describe('Members Route', function () {
+xdescribe('Members Route', function () {
```

Comment:
This test was already there. It isn't relevant yet, but I think we'll need it later, so I marked it pending.